### PR TITLE
DOC: add Examples and Returns in scipy.misc.central_diff_weights docstring.

### PR DESCRIPTION
**Pull request by AtsushiSakai**
#### Reference issue
fix a part of #7168

#### What does this implement/fix?
DOC: add Examples and Returns in scipy.misc.central_diff_weights.

### CODE REVIEW

File: scipy/misc/common.py [hunk 1]
```diff
@@ -25,10 +25,37 @@ def central_diff_weights(Np, ndiv=1):
...
+    We can calculate a derivative value of a function.
+
+    >>> from scipy.misc import central_diff_weights
+    >>> def f(x):
+    ...     return 2*x**2+3
+    >>> x = 3.0 # derivative point
+    >>> h = 0.1 # differential step
+    >>> Np = 3 # point number for central derivative
+    >>> weights = central_diff_weights(Np) # weights for first derivative
+    >>> vals = [f(x+(i-Np/2)*h) for i in range(Np)]
```
Comment (andyfaff) [inline scipy/misc/common.py:48]: PEP8:
```
>>> vals = [f(x + (i - Np/2) * h) for i in range(Np)]
```

File: scipy/misc/common.py [hunk 2]
```diff
@@ -25,10 +25,37 @@ def central_diff_weights(Np, ndiv=1):
...
     -----
     Can be inaccurate for a large number of points.
 
+    Examples
+    --------
+    We can calculate a derivative value of a function.
+
+    >>> from scipy.misc import central_diff_weights
+    >>> def f(x):
+    ...     return 2*x**2+3
```
Comment (andyfaff) [inline scipy/misc/common.py:43]: PEP8:
```
...     return 2 * x**2 + 3
```

**Review by andyfaff — CHANGES_REQUESTED** [review]

File: scipy/misc/common.py (hunk 1 above)
Comment (AtsushiSakai) [reply scipy/misc/common.py]: Thank you. I fixed it.

File: scipy/misc/common.py (hunk 2 above)
Comment (AtsushiSakai) [reply scipy/misc/common.py]: Thank you. I fixed it.

**Review by andyfaff — APPROVED** [review]
I'll squash and merge once the CI goes green.

**Comment by AtsushiSakai** [comment]
I’m sorry I miss push the button for requesting review. Please ignore it.

**Comment by AtsushiSakai** [comment]
I closed and reopen this PR to rerun CI.

**Pull request merged**